src/golang/lib/databricks/constants.go

```diff
@@ -0,0 +1,12 @@
+package databricks
+
+const (
+	SparkVersion = "10.4.x-scala2.12"
```

Comment:
i'd suggest to use dynamic selectors: https://github.com/databricks/databricks-sdk-go#node-type-and-databricks-runtime-selectors

also DBR 10.x is a bit old now.